app/src/main/java/org/grapheneos/apps/client/di/HttpModule.kt

```diff
@@ -3,11 +3,13 @@ package org.grapheneos.apps.client.di
 import androidx.annotation.Nullable
 import org.grapheneos.apps.client.item.Progress
 import org.grapheneos.apps.client.item.network.Response
+import org.grapheneos.apps.client.utils.network.Encoding
 import java.io.File
 import java.io.FileOutputStream
 import java.net.HttpURLConnection
 import java.net.URL
-import java.util.Locale
+import java.util.*
```

Comment:
Glob imports aren't allowed here. Needs to be replaced with `java.util.Locale` and/or any other stuff from there.

For some reason Android Studio will use `java.util.*` instead of just properly getting what you need.

Reply:
done.